pass/disk.go

```diff
@@ -42,7 +47,55 @@ func defaultStorePath() (string, error) {
 	return filepath.EvalSymlinks(path)
 }
 
+// Set the configuration options for password matching.
+func (s *diskStore) SetConfig(path *string, use_fuzzy *bool) error {
```

Comment:
I don't see any place where `path` is not nil, is this an obsolete parameter?

Reply:
Oh thanks for calling this out, I forgot to comment on it!

I have that in place because it feels like the Right Thing to have the path to password store be changeable via the `SetConfig` method. I went back and forth when writing it because there's two ways to view the semantics of `SetConfig`:
* `SetConfig` updates any settings that affect the password store wrapper (therefore keep `path`)
* `SetConfig` applies any app settings (therefore remove `path` for now - but i'm thinking of tackling #98 next which will re-introduce this parameter if removed)

That said I have no strong preference on it in terms of this PR.

Reply:
Let's leave it out for this PR and introduce in a separate one, just for clarity in git history.

#98 is a good issue to tackle next, actually there was #77 (which now opens 404 page for me? 😕 ), if I remember correctly it suggested to allow configuring multiple password store directories (e.g. one for work and one personal), and even toggle the list in options (i.e. temporarily disable work password store without removing it from options page). Something like this:

---------------

Password store directories:

- [x] `~/.password-store`
- [x] `~/.password-store-work` 

<kbd>Add</kbd>

--------------

 Anyhow, this we can discuss separately, I just wanted to let you know 🙂

Reply:
OK sounds good. FYI I'm also getting a 404 for #77 oddly. 

I'll move discussion of password dir settings to #98 when I dive into that.

Reply:
@sophacles just curious, do you think you will have time to work on #98? No pressure! 😄